### DISCUSSION TITLE
Using Fooocus Wildcards (Comprehensive Video Guide)

### DISCUSSION THREAD

**Discussion by ItsNoted** [lllyasviel/Fooocus#1667] [Show and tell]
I created this video that shows how to use wildcards in Fooocus. Hope you enjoy!

https://www.youtube.com/watch?v=E_R7tnfXKCM

**Comment by PeterJared** [comment]
That is a extremely useful video. Thanks for sharing!

**Comment by mashb1t** [comment]
Great work, nice video! When https://github.com/lllyasviel/Fooocus/pull/1503 is merged you may also include this information in the video description :)

**Reply by ItsNoted** [reply]
This is going to be amazing. Any idea when the merge will be? Also, any way to set the image number to 0 so we can generate infinitely until stopped manually?

**Reply by mashb1t** [reply]
Not exactly, also waiting for some other PRs to be reviewed. Probably mid/end of January as announced in the update log.

**Comment by Amit30swgoh** [comment]
> I created this video that shows how to use wildcards in Fooocus. Hope you enjoy!
> 
> https://www.youtube.com/watch?v=E_R7tnfXKCM

I recommend. great voice great vid ty again for helping
